python/mxnet/ndarray/numpy/random.py

```diff
@@ -129,6 +138,47 @@ def normal(loc=0.0, scale=1.0, size=None, **kwargs):
     Notes
     -----
     This function currently does not support ``loc`` and ``scale`` as ndarrays.
+    The probability density for the Gaussian distribution is
+
+    .. math:: p(x) = \frac{1}{\sqrt{ 2 \pi \sigma^2 }}
+                     e^{ - \frac{ (x - \mu)^2 } {2 \sigma^2} },
+    
+    where :math:`\mu` is the mean and :math:`\sigma` the standard
+    deviation. The square of the standard deviation, :math:`\sigma^2`,
+    is called the variance.
+    
+    The function has its peak at the mean, and its "spread" increases with
+    the standard deviation (the function reaches 0.607 times its maximum at
+    :math:`x + \sigma` and :math:`x - \sigma` [2]_).  This implies that
+    `numpy.random.normal` is more likely to return samples lying close to
+    the mean, rather than those far away.
+    
+    This function differs from the original numpy.random.normal in the following aspects:
+    
+        - Argument must be named from the fourth.
```

Comment:
This is not very clear to the users, I think you can emphasize that the order of arguments is different.